css/src/tokens/misc.scss

```diff
@@ -0,0 +1,33 @@
+// Radius
+$radius-small: 2px !default;
+$radius: 4px !default;
+$radius-large: 6px !default;
+$radius-rounded: 290486px !default;
+
+// Direction
+$user-text-direction: ltr !default;
+$content-text-direction: ltr !default;
+
+$user-left: if($user-text-direction == rtl, right, left);
+$user-right: if($user-text-direction == rtl, left, right);
+
+$content-left: if($content-text-direction == rtl, right, left);
```

Comment:
This wasn't in any of the tasks, so apologies about the churn: Atlas isn't going to support content directions. We can the content- related vars. The rest is looking good.

Reply:
Removed content- tokens.